jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -663,13 +657,6 @@ def user_api(self):
 
         return PluginUserApi(self, expose)
 
-    @observe('show_viewer_labels')
-    def _on_show_viewer_labels_changed(self, event):
-        self.app.state.settings['viewer_labels'] = event['new']
-
-    def _on_app_settings_changed(self, value):
-        self.show_viewer_labels = value['viewer_labels']

```

Comment:
this was never exposed in the user API, so probably doesn't need to be deprecated?  Although we may want to mention in explicitly in the changelog (hiding viewer labels is now longer supported since they will soon become the trigger for opening the data menu).